tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -3012,6 +3012,10 @@ public function adminCreatesGroupUsingTheProvisioningApi(string $group):void {
 	public function groupHasBeenCreated(string $group):void {
 		$this->createTheGroup($group);
 		$this->groupShouldExist($group);
+		Assert::assertTrue(
```

Comment:
you can remove `$this->groupShouldExist($group);`
```suggestion
		Assert::assertTrue(
```

Reply:
okay